src/ansys/units/systems.py

```diff
@@ -115,7 +115,7 @@ def BASE_UNITS_LENGTH(cls, len):
     @classmethod
     def UNIT_UNDEFINED(cls, unit):
         return cls(
-            f"`{unit}` is an undefined unit. To use `{unit}` add it to the `fundamental_units` table within quantity_config.yaml."
```

Comment:
@acarvalh-work somehow this branch is failing the unit tests. Not noticing such failures generally

Reply:
Oh, I see now, the test is asserting on the error content

Reply:
yes should be good now. There is a doc build issues related to verifying links (https://pypi.org/project/ansys-units) and (https://pyunits.docs.pyansys.com/). Working on this at the moment.